be2-scala/src/main/scala/ch/epfl/pop/storage/DbActor.scala

```diff
@@ -251,9 +251,9 @@ final case class DbActor(
         case failure              => sender() ! failure.recover(Status.Failure(_))
       }
 
-    case ReadElectionData(electionId) =>
+    case ReadElectionData(electionId, laoId) =>
```

Comment:
Here your ReadElectionData paramters order does not match the definition (See line 394). You need to switch the order of laoId and electionId